README.md

```diff
@@ -756,6 +757,38 @@ Example `makes.nix`:
 
 Example invocation: `$ m . /lintBash`
 
+### lintClojure
+
+Lints clojure code with [clj-kondo][CLJ-KONDO].
+
+Attributes:
+
+- targets (`attrsOf (listOf str)`): Optional.
+  Mapping of custom names to lists of paths (relative to the project) to lint.
+  Defaults to `{ }`.
+
+Example `makes.nix`:
+
+```nix
+{
+  lintClojure = {
+    targets = {
+      example1 = [
+        "/" # Entire project
+        "/file.py" # A file
+      ];
+      example2 = [
+        "/directory" # A directory within the project
+      ];
+    };
+  };
+}
+```
+
+Example invocation: `$ m . /lintClojure/example1`
```

Comment:
you are not using `targets`, so we can remove it

```nix
{
  lintClojure = {
      example1 = [
        "/" # Entire project
        "/file.py" # A file
      ];
      example2 = [
        "/directory" # A directory within the project
      ];
    };
}
```

Reply:
same for lintWithLizard?

Reply:
would be a nice idea for simplicity

Reply:
on it